h2o-hadoop/h2o-cdh6.0/CDH60.Makefile

```diff
@@ -0,0 +1,100 @@
+CDHV=6.0
+#CDHV=5.14
+
+
+all: build
+
+clean:
+	./gradlew clean
+
+build: .PHONY
+	./gradlew build -x test
+
+run:
+	java \
+		-Djetty.request.header.size=32768 \
```

Comment:
why is this necessary?

Reply:
Whole file is a leftover, will be deleted. I copy&pasted past changes and made changes on my own in the past commits. Forgot about this file.